docs/architecture/specs/examples/access/v0.1/ddo_nft.json

```diff
@@ -117,9 +131,7 @@
 					"ercType": 1155,
 					"name": "nftSalesAgreement",
 					"creator": "",
-					"datePublished": "2019-02-08T08:13:49Z",
-					"price": "20",
```

Comment:
I realize now that removing `price` here will create a problem because now we have now way to query for price because we only have an array `amounts`.

My proposal is to remove price from the metadata attributes but leave it on the services with a LockPaymentCondition.

Additional proposal related with https://github.com/nevermined-io/marketplace-api/issues/94:
We always know the tokenAddress being used and so we can get the decimals. I propose use `main.attributes.price` to store the total price in the highest denomination (eth, usdc, ..) as a float. That way we can get price query range working correctly. While developing carbon-ox I used this work arround by adding a usdcprice field in the additional information

Reply:
looks good